code/game/turfs/turf.dm

```diff
@@ -241,12 +240,63 @@
 
 		C << "<span class='notice'>You slipped[ O ? " on the [O.name]" : ""]!</span>"
 
+		if(C.dna)
+			if(C.dna.features["tail_human"] == "Cat" && C.dna.features["ears"] == "Cat")
+				C.SpinAnimation(7,1)
+				visible_message("<span class='notice'>[C.name] slips... but manages to stick the landing with their cat like reflexs!</span>")
+				return 0
+		var/list/impact_point = pick("Head", "Chest", "Back", "Ass", "Hands", "Groin")
+		var/message
+		switch(impact_point)
+			if("Head") //oh boy
+				message = "[C.name] slips backwards straight onto their head! You hear a"
```

Comment:
just a remnant, shouldn't ever be seen in game but will remove if i have to change anything else.